packages/sit-onyx/src/components/OnyxRadioButton/OnyxRadioButton.stories.ts

```diff
@@ -3,13 +3,13 @@ import type { Meta, StoryObj } from "@storybook/vue3";
 import OnyxRadioButton from "./OnyxRadioButton.vue";
 
 /**
- * The input component can be used to...
+ * This is a support component for the `OnyxRadioButtonGroup`.
  */
 const meta: Meta<typeof OnyxRadioButton> = {
   title: "support/OnyxRadioButton",
   ...defineStorybookActionsAndVModels({
     component: OnyxRadioButton,
-    events: [],
+    events: ["change"],
```

Comment:
The component does not have a change event defined with defineEmits

Reply:
This is the native `change` event

Reply:
Yes but we should still define it so it is included and documented in the Storybook events

Reply:
Otherwise people will not know that it exists

Reply:
I don't think we should document native events, e.g. we would need to document `click` on every component.
The event was only added here so that you can see something happening in the actions.